toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -41,6 +49,10 @@ const (
 	dracutConfig = `add_dracutmodules+=" dmsquash-live "
 add_drivers+=" overlay "
 `
+	// the total size of a collection of files is multiplied by the
+	// expansionSafetyFactor to estimate a disk size sufficient to hold those
+	// files.
+	expansionSafetyFactor = 1.5
```

Comment:
curious - is this based of anything specific or just a buffered estimate?

Reply:
not based on anything specific - just a buffered estimate.
I've found out that `du` returns size in disk blocks - not in actual file contents. So, `du` will return 4k for 2 byte file, for example.  This makes me think we can lower the safety factor, since choosing a block size that's not 4k is not common. Open to suggestions here...